tools/docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019 by Cornel Izbasa. All rights reserved.
```

Comment:
I guess this is you @Mr-Slippery? I need to confirm that we have the approval from the Author to use this codesnippet.

Reply:
Hi @dkroenke, yes, it's just my Github username. :) The Eclipse account linked to this Github username has my real name, Cornel Izbasa. Thanks!

Reply:
Are you maybe located in Berlin? I found a LinkedIn Profile which says that you are in Tempelhof.

Reply:
I work at HELLA in Tempelhof, but live in North-East of Berlin :)